resources/migrations/20211118093519-create-user.up.sql

```diff
@@ -0,0 +1,12 @@
+DROP TYPE IF EXISTS role;
```

Comment:
I & @alishamohanty did this because we wanted to discard `role` if it already exists.
But if `role` already exists then we think it is an unrelated bug which should not be fixed using another `CREATE TYPE` command. Will brief more in the evening catchup